### DISCUSSION TITLE
Updated Raycast Integration

### DISCUSSION THREAD

**Discussion by pomdtr** [johnlindquist/kit#607] [Share]
I've built a new extension to be able to launch my scripts from raycast. It would be cool to get a review from you @johnlindquist !

Link to the pull request: https://github.com/raycast/extensions/pull/399

https://user-images.githubusercontent.com/17577332/142733677-ae7fcd29-e267-4301-a4e7-1ac86b030c98.mp4

**Comment by johnlindquist** [comment]
Oooooooh, nice. I'll dive in next week! Trying to get a build out today w/ better docs, settings, and minor fixes then start recording lessons on Monday 😳

**Reply by johnlindquist** [reply]
Nevermind, that was easy to review. Nice work!

**Reply by pomdtr** [reply]
Thanks a lot!

**Reply by pomdtr** [reply]
Feel free to showcase the integration in your docs once the extension will be published on the raycast store !

**Comment by pomdtr** [comment]
Small issue I did encounter when creating the extension: the kar cli is not able to resolve the path from any script in the example kenv, so I switched to the full path of the script instead. It should be easy to fix on your side :)

**Reply by johnlindquist** [reply]
Yeah, Kenvs can have scripts with the same names, so the full path is required.

kit/kar is almost exclusively used for launching from other apps, so you only have to type out the full path when you initially set it up.

In contrast, kit/bin/kit will attempt to find scripts relatively based on where you invoke it from in the terminal.

**Reply by pomdtr** [reply]
I guess it would have been less confusing for me if kar did not allow me to launch my own scripts without providing the full path.

**Reply by johnlindquist** [reply]
Yeah, kar existed way before kenvs...

I'm sure more kenv gotchas will pop-up once I put out more materials on how to use them and more community kenvs get shared.
